### PR TITLE
chore(repo): Add GH issue template

### LINKED ISSUE #2952: Use new Github issue templates
Yeeeessss!

https://github.com/blog/2111-issue-and-pull-request-templates

### PR DESCRIPTION
**Pull request by NickTomlin**
I took a first crack at creating an issue template (so excited about that feature!). I am not attached to anything here so fire away.

Submitting an issue:

screen shot 2016-02-17 at 4 51 07 pm

Submitted Issue:

screen shot 2016-02-17 at 4 52 42 pm

(this Closes #2952 )

### CODE REVIEW

File: .github/ISSUE_TEMPLATE [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+1. Review the questions section of [CONTRIBUTING.md](https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#contributing)
+2. Ensure that your issue is a bug or a feature request, and not a usage or debugging question
+3. Fill in the information below:
+4. Delete this intro text :smile:
+
+---
+
+- Node Version: `4.2.3`
+- Protractor Version: `3.1.1`
+- Operating System and Version `Mac Os X 10.10`
```
Comment (juliemr) [inline .github/ISSUE_TEMPLATE]: Maybe add a `browser` field?

File: .github/ISSUE_TEMPLATE [hunk 2]
```diff
@@ -0,0 +1,18 @@
+Hi there!
+
+Thanks for submitting an issue to Protractor.
+
+To help us help you better, please do the following before submitting an issue:
+
+1. Review the questions section of [CONTRIBUTING.md](https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#contributing)
+2. Ensure that your issue is a bug or a feature request, and not a usage or debugging question
+3. Fill in the information below:
+4. Delete this intro text :smile:
```
Comment (juliemr) [inline .github/ISSUE_TEMPLATE]: Would it be terrible to add something like "If you don't delete the intro text, we'll assume you didn't read the instructions and auto-close your issue"?

File: .github/ISSUE_TEMPLATE [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+
+---
```
Comment (juliemr) [inline .github/ISSUE_TEMPLATE]: What about having two sections, one for Feature Request and one for Bug Report?

**Comment by juliemr** [comment]
Thanks for getting this started! Added a couple first thoughts, hopefully others will chime in as well.

File: .github/ISSUE_TEMPLATE (hunk 3 above)
Comment (NickTomlin) [inline .github/ISSUE_TEMPLATE]: Great idea. I've updated with a spike of a pull request section.

File: .github/ISSUE_TEMPLATE (hunk 2 above)
Comment (NickTomlin) [inline .github/ISSUE_TEMPLATE]: Not at all, I think that makes a lot of sense

File: .github/ISSUE_TEMPLATE (hunk 1 above)
Comment (NickTomlin) [inline .github/ISSUE_TEMPLATE]: done!

File: .github/ISSUE_TEMPLATE [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+
+Bug report
+---
+
+- Node Version: ``
+- Protractor Version: ``
+- Browser(s): ``
+- Operating System and Version ``
+- Your protractor configuration file
+- A relevant example test
```
Comment (sjelin) [inline .github/ISSUE_TEMPLATE]: If possible/relevant, the webpage they're running their tests against

File: .github/ISSUE_TEMPLATE [hunk 5]
```diff
@@ -0,0 +1,30 @@
+Hi there!
+
+Thanks for submitting an issue to Protractor.
+
+To help us help you better, please do the following before submitting an issue:
+
+1. Review the questions section of [CONTRIBUTING.md](https://github.com/angular/protractor/blob/master/CONTRIBUTING.md#contributing)
+2. Ensure that your issue is a bug or a feature request, and not a usage or debugging question
```
Comment (sjelin) [inline .github/ISSUE_TEMPLATE]: Add "Ask debugging questions on [StackOverflow](http://stackoverflow.com/questions/tagged/protractor), [Google Group discussion list](https://groups.google.com/forum/?fromgroups#!forum/angular), or [Gitter](https://gitter.im/angular/protractor)."

Comment (sjelin) [inline .github/ISSUE_TEMPLATE]: There are other categories than bug reports and feature request (e.g. documentation).  Maybe just say "Make sure you are not asking a debugging question.  Ask those on StackOverflow, our Google Group discussion list, or Gitter."

**Comment by sjelin** [comment]
Love this.  Just a few small notes.  Great as is though

**Comment by NickTomlin** [comment]
@sjelin sorry for the late response, i've just updated this per your feedback. Thanks for the :eyes:

**Comment by juliemr** [comment]
Merged as 3bc465cbfe488577b59e27689f0581ae52620631

Yay!

**Comment by NickTomlin** [comment]
:confetti_ball:

**Pull request closed without merging**